submissions/ArthurGorbenko/html.css.practice/README.md

```diff
@@ -0,0 +1,2 @@
+## HTML CSS POPUP
+[demo](https://arthurgorbenko.github.io/Html_css_popup/)
```

Comment:
Thank you for your attention to details! I all time worked with repo.

Reply:
folder `images` should be in the same directory as `css` folder, do in your root direcotiry shoud be in total: 
* index.html
* readme.md
* css (folder)
* images (folder)

Reply:
Try to make your commits to have a self-description.
Example: commit name `project refactored`  will be better `moved filder img to css folder'.

You can enrich description for commit with command `git commit`:
* will be opened built-in terminal nano (if not nano, make nano default instead of vim)
* the first line will be commit name
* make one empty line and multiline description to the commit
* press `CTRL+O` to write out
* press `enter` to save 
* press `CTRL+X` to exit the editor

Reply:
1.I agree, that it has more sense.[Moved](https://github.com/ArthurGorbenko/Html_css_popup/commit/b75b63ec0b8ccbdd1a0ada362a47ec8a3729530f).
2. My fault, forgot about description.Btw, i have set as default VSCode, for me it is more practical.